python/lsst/analysis/tools/atools/calexpMetrics.py

```diff
@@ -35,39 +35,41 @@ class CalexpSummaryMetrics(AnalysisTool):
 
     # raCorners and decCorners statistics cannot be written to a metric,
     # as metrics can only be single-valued (i.e., scalars).
+    # Units in comments are to indicate compound units, which are currently
+    # unsupported.
     _units = {
         "psfSigma": "pixel",
-        "psfArea": "",  # There is no astropy sq. pixel unit.
-        "psfIxx": "pixel",
-        "psfIyy": "pixel",
-        "psfIxy": "pixel",
+        "psfArea": "",  # pixel**2
+        "psfIxx": "",  # pixel**2
+        "psfIyy": "",  # pixel**2
+        "psfIxy": "",  # pixel**2
         "ra": "degree",
         "dec": "degree",
-        "pixelScale": "",  # There is no astropy arcsec/pixel unit.
+        "pixelScale": "",  # arcsec/pixel.
         "zenithDistance": "degree",
         "expTime": "s",
         "zeroPoint": "mag",
-        "skyBg": "mag",
-        "skyNoise": "mag",
-        "meanVar": "mag",
+        "skyBg": "electron",
+        "skyNoise": "electron",
+        "meanVar": "",  # electron**2
         "astromOffsetMean": "arcsec",
         "astromOffsetStd": "arcsec",
-        "nPsfStar": "",
-        "psfStarDeltaE1Median": "pixel",
-        "psfStarDeltaE2Median": "pixel",
-        "psfStarDeltaE1Scatter": "pixel",
-        "psfStarDeltaE2Scatter": "pixel",
+        "nPsfStar": "ct",
```

Comment:
While I originally thought this usage of "count" was correct, on reading more documentation it seems like it is used more as a photometric unit, so it might make more sense to leave this unitless. Here is some usage I found [in the astropy doc](https://docs.astropy.org/en/latest/units/ref_api.html#module-astropy.units.function.units) and some [remarks in the iova doc](https://www.ivoa.net/documents/VOUnits/20231215/REC-VOUnits-1.1.html#sec:other).  I'd be interested in your opinion.

Reply:
Yeah - this has crossed my mind in the past. I've used it in other places to refer to things that are unitless integer and "counting". GIven that I (and others) have used it in other places with this meaning, I'd like to keep it here. However, I think it would be useful to do an audit of atools to make sure that it is used consistently.

Reply:
Sounds good to me!